counter/src/boot.js

```diff
@@ -1 +1 @@
-require("./Main.purs").main()
\ No newline at end of file
+require("~/output/Main").main()
```

Comment:
TIL: https://parceljs.org/module_resolution.html#module-resolution

Reply:
Minor: To appeal to current JS developers, we might want to prefer `import` over `require` here.

Reply:
++

Reply:
I am frankly so sick and tired of having up appeal to cargo-cult developers, I can't even...

Reply:
<understand sarcasm, choosing to respond seriously because I think it's actually relevant>
A cargo cult is when you have no idea why the vast majority are doing a thing, you see them deriving a benefit, and you ascribe that benefit to an ancillary feature -- eg, the islands around us are becoming wealthy, they have military planes landing there, military planes need runways, therefore if we build a runway we'll be wealthy.  

Appealing to current JS developers isn't that.  It's the opposite.  Choosing the alternative (without compelling reason for why it's better) is choosing to be the isolate.

</understanding....usually.>

Reply:
I do have a reason for choosing `require` over `import` - it's because I consider this module part of FFI (it only exists to kick the PureScript entry point), and the choice of `require` is consistent with the other place I have FFI - `src/Counter/Counter.js` - which has to be that way for technical reasons.

On the other hand, the hypothetical JS developer to whom I'm supposed to appeal either doesn't understand that `require` and `import` are the same thing, and insists on `import`, because that's what Facebook docs say (i.e. "cargo cult"); or else does understand, and yet somehow holds my code in lower regard despite the understanding, which is not quite cargo cult, but still very irrational.